tests/cypress/fixtures/marketplace.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
this file isn't needed anymore, there's now a `products.json` file that is loaded in the test.